app/dashboard/templates/profiles/profile.html

```diff
@@ -234,10 +235,32 @@
                       </div>
                     </div>
                   </div>
-                </div>
-                {% endif %}
+                </div> {% endif %}
               </div>
             {% endfor %}
+		{% else %}
+		<BR>
+		<div class="row" style="justify-content: center"><img width="240px" height="225px" src="http://kuhnchris.eu:8000/static/v2/images/tree.png" /></div><BR>
+		<div style="text-align: center; color: #666666; font-size: 16px">Oops! Looks like this user has not participated in any bounty yet.<BR>In the meantime, here are some things you can do:</div><BR>
+		<div class="row" style="justify-content: center">
+			<div style="width: 25%;justify-content: center;margin-right: 1em; padding: 1em; margin-top: 1em;"  class="card col-6 col-lg-3">
+
+					<img style="margin: 1em;" src="{% static "v2/images/IssueExplorer.svg" %}"/>
+        			<div class="description" style="color: #666666; text-align: center">Start working on exciting bounties with the Issue Explorer</div>
+       				<a class="btn btn-xs btn-success pulseClick" href="{% url "explorer" %}" style="background-color: #3E00FF; color: #FFFFFF; margin: 1em 0em 1em 0em; font-size: 1em	">Go to Issue Explorer</a>
+    		</div>
+			<div style="width: 25%;justify-content: center;margin-right: 1em; padding: 1em; margin-top: 1em;"  class="card col-6 col-lg-3">
```

Comment:
Can you try to use our existing CSS rules where possible? (`app/assets/v2/css`)
We also include bootstrap.css, which means we have something like `m-{1,2..}` for margin and 'p-{1,2,3..}` for padding etc.
 👍

Reply:
If we have several predefined classes for margin or padding then, sure, I'll change that. Want me to add the other stuff in custom classes in assets/v2/css or just trying to reduce the amount of style use as much as possible?

Reply:
Just trying to reduce the use of the style attribute ;).
If the existing CSS rules doesn't fit your needs, you can of course use the style attribute or add it to the existing `profile.css` 👍